applications/matter_bridge/README.rst

```diff
@@ -383,6 +388,21 @@ After building the sample and programming it to your development kit, complete t
 
                ./chip-tool onoff read on-off *<bridge_node_ID>* *<bridged_device_endpoint_ID>*
 
+            In case of the Environmental Sensor, the current temperature and humidity measurements forwarded by the Bluetooth LE Environmental Sensor can be read as follows:
+              * temperature:
+
+            .. parsed-literal::
+               :class: highlight
+
+               ./chip-tool temperaturemeasurement read measured-value *<bridge_node_ID>* *<bridged_device_endpoint_ID>*
+
+              * humidity:
+
+            .. parsed-literal::
+               :class: highlight
+
+               ./chip-tool relativehumiditymeasurement read measured-value *<bridge_node_ID>* *<bridged_device_endpoint_ID>*
```

Comment:
```suggestion

              * temperature:

                .. parsed-literal::
                   :class: highlight

                   ./chip-tool temperaturemeasurement read measured-value *<bridge_node_ID>* *<bridged_device_endpoint_ID>*

              * humidity:

                .. parsed-literal::
                   :class: highlight

                    ./chip-tool relativehumiditymeasurement read measured-value *<bridge_node_ID>* *<bridged_device_endpoint_ID>*
```
Empty line before the first bullet, and correct indentation for the parsed-literal entries.